<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2200</width>
    <height>1066</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(120, 120, 120);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(167, 167, 167);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <layout class="QGridLayout" name="main_gride">
         <item row="0" column="2">
          <widget class="QFrame" name="ep_frame">
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <layout class="QGridLayout" name="ep_gride">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item row="2" column="0">
               <widget class="QTableWidget" name="ep_table">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>500</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:(255,255,255)</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="showGrid">
                 <bool>true</bool>
                </property>
                <property name="gridStyle">
                 <enum>Qt::DotLine</enum>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="ep_score">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <layout class="QVBoxLayout" name="ep_predict_figure">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QVBoxLayout" name="ep_features_figure">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="ep_title">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>22</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>EP</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QFrame" name="mri_frame">
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="0" column="0">
             <layout class="QGridLayout" name="mri_gride">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item row="4" column="0">
               <layout class="QVBoxLayout" name="mri_predict_figure">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QVBoxLayout" name="mri_features_figure">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="mri_score">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QTableWidget" name="mri_table">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>500</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:(255,255,255)</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="showGrid">
                 <bool>true</bool>
                </property>
                <property name="gridStyle">
                 <enum>Qt::DotLine</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="mri_title">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>22</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>MRI</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer_12">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QFrame" name="eeg_frame">
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_16">
            <item row="0" column="0">
             <layout class="QGridLayout" name="eeg_gride">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item row="1" column="0">
               <spacer name="verticalSpacer_11">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="eeg_title">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>22</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>EEG</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QTableWidget" name="eeg_table">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>500</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:(255,255,255)</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="showGrid">
                 <bool>true</bool>
                </property>
                <property name="gridStyle">
                 <enum>Qt::DotLine</enum>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="eeg_score">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <layout class="QVBoxLayout" name="eeg_features_figure">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </item>
              <item row="4" column="0">
               <layout class="QVBoxLayout" name="eeg_predict_figure">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="vr_frame">
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_18">
            <item row="0" column="0">
             <layout class="QGridLayout" name="vr_gride">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item row="1" column="0">
               <spacer name="verticalSpacer_13">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="vr_title">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>22</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>VR</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QTableWidget" name="vr_table">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>500</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:(255,255,255)</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="showGrid">
                 <bool>true</bool>
                </property>
                <property name="gridStyle">
                 <enum>Qt::DotLine</enum>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="vr_score">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <layout class="QVBoxLayout" name="vr_features_figure">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </item>
              <item row="4" column="0">
               <layout class="QVBoxLayout" name="vr_predict_figure">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="3" colspan="2">
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="1" column="1">
             <layout class="QGridLayout" name="mri_result"/>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="eeg_result"/>
            </item>
            <item row="0" column="1">
             <layout class="QGridLayout" name="ep_result"/>
            </item>
            <item row="0" column="0">
             <layout class="QGridLayout" name="vr_result"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="3" colspan="2">
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="4" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="ep_title_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>22</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>REPORT</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="0">
               <layout class="QGridLayout" name="total_figure"/>
              </item>
              <item row="2" column="0">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2200</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPrev">
    <property name="title">
     <string>Prev</string>
    </property>
   </widget>
   <addaction name="menuPrev"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
